.pre-commit-config.yaml

```diff
@@ -165,6 +156,12 @@ repos:
         hooks:
             - id: rapids-dependency-file-generator
               args: ["--clean"]
+      - repo: https://github.com/astral-sh/ruff-pre-commit
+        rev: v0.0.278
+        hooks:
+          - id: ruff
+            files: python/.*$
```

Comment:
I’d be open to running ruff on the whole repo if that makes sense. It could clean up various helper scripts, for example. Maybe in a follow-up.

Reply:
Agreed, I would like to do that but for this PR kept the scope limited.